lib/node_modules/@stdlib/math/base/special/sincospi/README.md

```diff
@@ -0,0 +1,78 @@
+# sincospi
+
+> Simultaneously compute the [sine][@stdlib/math/base/special/sin] and [cosine][@stdlib/math/base/special/cos] of a number times [π][@stdlib/math/constants/pi].
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var sincospi = require( '@stdlib/math/base/special/sincospi' );
+```
+
+#### sincospi( \[out,] x )
+
+Simultaneously computes the [sine][@stdlib/math/base/special/sin] and [cosine][@stdlib/math/base/special/cos] of a `number` times [π][@stdlib/math/constants/pi] more accurately than `sincos(pi*x)`, especially for large `x`..
+
+```javascript
+var v = sincospi( 0.0 );
+// returns [ ~0.0, ~1.0 ]
```

Comment:
The return values need to be updated.

In that context, I recall you asking about possible automation of these rote tasks. I have a local addon for VSCode, which inserts return value annotations, but it works only in limited cases, so it's not really ready for prime-time. But we should probably aim for an editor-agnostic command-line application.

Reply:
Hmm… are they incorrect?

Reply:
No, all is fine. Didn't realize the input was zero. Sorry about the confusion I caused.